voc_builder/interactive.py

```diff
@@ -382,47 +382,49 @@ def handle_cmd_no() -> NoActionResult:
     if not choice:
         console.print('Skipped.', style='grey42')
         return NoActionResult(error='user_skip')
-
-    word_sample = WordSample(
-        word=choice.word,
-        word_normal=choice.word_normal,
-        word_meaning=choice.word_meaning,
-        pronunciation=choice.pronunciation,
+    
+    word_samplelist = []
+    for i in choice:
+        word_samplelist.append(WordSample(
+        word=i.word,
+        word_normal=i.word_normal,
+        word_meaning=i.word_meaning,
+        pronunciation=i.pronunciation,
         translated_text=ret.word_sample.translated_text,
         orig_text=ret.input_text,
-    )
-
-    try:
-        validate_result_word(word_sample, ret.input_text)
-    except WordInvalidForAdding as e:
-        console.print(f'Unable to add "{word_sample.word}", reason: {e}', style='grey42')
-        return NoActionResult(word=word_sample, stored_to_voc_book=False, error=str(e))
+    ))
+    
 
     word_store = get_word_store()
-    word_store.add(word_sample)
-    console.print(
-        (
-            f'[bold]"{word_sample.word}"[/bold] was added to your vocabulary book ([bold]{word_store.count()}[/bold] '
-            'in total), well done!\n'
-        ),
-        style='grey42',
-    )
+    for word_sample in word_samplelist:
```

Comment:
A better approach would be to try to save all the words first, print the successfully added words, and print the failed words. In this case, the printed message would be much simpler.